pkg/arvo/neo/cod/std/src/fil/developer-environment-setup.txt

```diff
@@ -0,0 +1,16 @@
+# Developer Environment Setup
+
+The quickest way to get set up to play with Shrubbery and Sky is to boot a comet and run `|install ~met %neo`.
+
+If you want to work from the GitHub repo…
+- Clone [urbit/shrub](https://github.com/urbit/shrub) to your machine. The `develop` branch is the latest stable build.
+- Boot a fakeship
+- `|new-desk %neo` on the fakeship
+- `|mount %neo`
+- `rsync -avL /path/to/pkg/shrub/* /path/to/fakeship/neo`
+- `|commit %neo`
+- `|install our %neo`
```

Comment:
Use prior letter to indicate in what environment a command is intended (in addition to any explicit verbal instructions)

`$`: bash/unix shell
`#`: bash/unix shell (as root)
`>`: dojo